test/lib/spender_web/controllers/page_controller_test.exs

```diff
@@ -5,7 +5,7 @@ defmodule SpenderWeb.PageControllerTest do
   describe "Page Controller" do
      test "renders the homepage on the root url", %{conn: conn} do
       conn = get conn, "/"
-      assert text_response(conn, 200)
+      assert html_response(conn, 200)
```

Comment:
since we know some of the text on the page we can make an asserting here that checks for the words "Welcome to the Spender API" on the page since we know that they should appear see this example on how to do this https://hexdocs.pm/phoenix/Phoenix.ConnTest.html#html_response/2-examples